File: csharp/src/TodoApi/Models/Todo.cs
using System;

namespace TodoApi.Models;

public class Todo
{
    public string Id { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public bool Completed { get; set; }
    public string UserId { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: go/internal/models/todo.go
package models

import "time"

type Todo struct {
	Id        string    `json:"id"`
	Title     string    `json:"title"`
	Completed bool      `json:"completed"`
	UserId    string    `json:"userId"`
	CreatedAt time.Time `json:"createdAt"`
	UpdatedAt time.Time `json:"updatedAt"`
}
